sdks/go/pkg/beam/core/runtime/harness/worker_status.go

```diff
@@ -104,7 +104,7 @@ func goroutineDump(statusInfo *strings.Builder) {
 func buildInfo(statusInfo *strings.Builder) {
 	statusInfo.WriteString("\n============Build Info============\n")
 	if info, ok := debug.ReadBuildInfo(); ok {
-		statusInfo.WriteString(info.String())
```

Comment:
In this case we prioritize the Open Source static check, and not something that only affects Google internal users on code import, where we can ignore them. This will cause the Github Go Tests action to fail for everyone.